web/src/components/Wrapper.test.js

```diff
@@ -32,7 +32,7 @@ describe('Wrapper component', () => {
   });
 
   describe('sets the appropriate CSS class if we are on a "gray" path', () =>
-    ['/create-account', '/edit-account', '/login', '/me'].forEach(pathname => {
+    ['/login'].forEach(pathname => {
```

Comment:
you could take the forEach out and just do this explicitly for /login since that's the only one being tested now

Reply:
good catch, not sure how I didn't see it :doh: